protokube/pkg/protokube/aws_volume.go

```diff
@@ -164,73 +164,83 @@ func newEc2Filter(name string, value string) *ec2.Filter {
 func (a *AWSVolumes) findVolumes(request *ec2.DescribeVolumesInput) ([]*Volume, error) {
```

Comment:
We may be talking in circles since from my perspective we are doing a post-filter in the code that I wrote and tested.  Sorry confused